build_fenics_deps.sh

```diff
@@ -1,26 +1,27 @@
 #!/bin/bash
 
-if [ ${CONDA_BUILD_TYPE} = host-gcc ]; then
+if [ "${CONDA_BUILD_TYPE}" = "host-gcc" ]; then
     # Dependencies
-#     conda build subprocess32/host-gcc
-#     conda build eigen3/host-gcc
-#     conda build boost/host-gcc
-#     conda build hdf5-parallel/host-gcc
-#     conda build h5py-parallel/host-gcc
-#     conda build vtk/host-gcc
+    conda build subprocess32/host-gcc
+    conda build eigen3/host-gcc
+    conda build boost/host-gcc
+    conda build hdf5-parallel/host-gcc
+    conda build h5py-parallel/host-gcc
+    conda build vtk/host-gcc
     conda build petsc/host-gcc
-#     conda build petsc4py/host-gcc
-#     conda build slepc/host-gcc
+    conda build petsc4py/host-gcc --no-remove-work-dir
```

Comment:
Her er feilmeldinga: ValueError: In conda-build 2.1+, the work dir is removed by default before the test scripts run.  You are using the SRC_DIR variable in your test script, but these files have been deleted.  Please see the  documentation regarding the test/source_files meta.yaml section, or pass the --no-remove-work-dir flag.